src/serverMain/kotlin/com/studo/campusqr/database/AutomaticCheckOut.kt

```diff
@@ -0,0 +1,39 @@
+package com.studo.campusqr.database
+
+import com.studo.campusqr.extensions.addMinutes
+import com.studo.campusqr.extensions.runOnDb
+import com.studo.campusqr.serverScope
+import com.studo.katerbase.equal
+import com.studo.katerbase.lower
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import java.util.*
+
+/**
+ * This background task deletes old [CheckIn]s after a defined amount of days
+ */
+fun startAutomaticCheckOut() = serverScope.launch {
```

Comment:
Is this function unused? @functionaldude

Reply:
fixed